src/actors/containerization/services/apache/apache_generator/apache_generator.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+"""
+Author: tmeszaro@redhat.com
+
+
+Apache generator actor. Uses data from apache_scanner, creates image with
+buildah and pushes this image into the specified registrty.
+
+
+Running the container:
+
+- after pulling image from registry, run:
+
+$ docker run -dt -p HOST_PORT:CONTAINER_PORT IMAGE_ID
+
+
+Possible improvements:
+
+- build from this? registry.access.redhat.com/rhscl/httpd-24-rhel7
+- parse httpd -S or httpd -V to get main config location, so we can run httpd
+   with non-standard config location
+- copy also other things (like logs, etc.)
+- move image pushing into separate actor
+
+"""
+
+
+import json
+import os
+import sys
+import subprocess
+import shlex
+from distutils.spawn import find_executable
+
+
+def find_buildah():
```

Comment:
We shouldn't do it here, other actor should check installed pkgs/binaries. I know it doesn't exist now, but i think this should be removed, just make this assumption that builda exists.

Reply:
Ok, my bad, the main purpose of this it's to find buildah :) (ofc func name is saying that :) )